src/languages/sqf.js

```diff
@@ -2473,6 +2473,6 @@ export default function(hljs) {
       STRINGS,
       PREPROCESSOR
     ],
-    illegal: /^\$ /
+    illegal: /\w\$|\?|\@/
```

Comment:
Please break these out into an array and add a comment explaining why each of these is illegal to help future readers/maintenance.

Reply:
@joshgoebel is this ok?
```js
illegal: [
      '\w\$',  //$ is only valid when used with Hex numbers (e.g. $FF) or at the beginning of localized strings (e.g. "$STR_HELLO")
      '\?',    //There's no ? in SQF
      '\@/'    //There's no @ in SQF
    ]
```